src/View/Rooms/index.html.twig

```diff
@@ -2,7 +2,7 @@
 {% block title %}Chambres Grand Hôtel{% endblock %}
 {% block content %}
     <h1>Les chambres du Grand Hôtel d'Orléans</h1>
-           {% for key, room in rooms %}
+    {% for key, room in rooms %}
         <div class="card roomCard mb-3 pb-3 mt-4">
             <div class="row m-4 align-items-start CarouselPageChambres{% if key is even %}d-flex flex-row-reverse {% endif %}">
```

Comment:
le key me semble inutile ? 
un loop.index is even est suffisant je pense

Reply:
J'ai mis le loopindex even à la place du key.
Mais je garde le key : je l'utilise ligne 23 pour boucler sur le tableau caractéristiques